### PR TITLE
Cleaning up the tests

### PR DESCRIPTION
**Pull request by ameech**

### CODE REVIEW

File: tests/RouterTest.php [hunk 1]
```diff
@@ -87,16 +87,23 @@ public function testDispatching()
         $methods = $this->routeMethodProvider();
         $router = Application::boot()->getRouter();
 
-        foreach ($methods as $data) {
-            $router->{$data[0]}('/test', '\SparkTests\Fake\FakeDomain');
+        $path = '/test';
```
Comment (ameech) [inline tests/RouterTest.php]: @shadowhand what do you think of this test now?

Comment (shadowhand) [inline tests/RouterTest.php]: :trophy:

**Comment by shadowhand** [comment]
Just as a side note, this commit message is not in the correct tense, it reads:

> [This commit will] cleaning up the tests

**Comment by ameech** [comment]
:+1: Thanks for the heads up.

**Pull request merged**